### PR TITLE
feat(a380x): Add ATSU / Connect to TELEX

### PR DESCRIPTION
**Pull request by flogross89**
## Summary of Changes

Adds ATSU to SystemsHost, connects to TELEX once FLT NBR on FMS/ACTIVE/INIT page has been filled

Connect, if FLT NBR is filled
Disconnect, if FLT NBR is erased

## Screenshots (if necessary)

image

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: fbw-a32nx/src/systems/fmgc/src/flightplanning/FpmConfig.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class FpmConfigs {
     TMPY_ON_INSERT_WAYPOINT: true,
     ALLOW_REVISIONS_ON_TMPY: true,
     ALLOW_NON_ACTIVE_FIX_INFOS: false,
-    MAX_NUM_LEGS: 200,
+    MAX_NUM_LEGS: 250,
```
Comment (tracernz) [inline fbw-a32nx/src/systems/fmgc/src/flightplanning/FpmConfig.ts:46]: This should be done in a different PR.

**Review by tracernz — CHANGES_REQUESTED** [review]

File: fbw-a380x/src/systems/systems-host/index.ts [hunk 2]
```diff
@@ -101,6 +104,7 @@ class SystemsHost extends BaseInstrument {
     this.backplane.addInstrument('Amu2', this.amu2, true);
     this.backplane.addInstrument('SimAudioManager', this.simAudioManager);
     this.backplane.addInstrument('Xpndr1', this.xpdr1, true);
+    this.backplane.addInstrument('AtsuSystem', this.atsu,  true);
```
Comment (tracernz) [inline fbw-a380x/src/systems/systems-host/index.ts:107]: Why do you want to allow duplicates?

**Review by tracernz — APPROVED** [review]

File: fbw-a380x/src/systems/systems-host/index.ts (hunk 2 above)
Comment (flogross89) [reply fbw-a380x/src/systems/systems-host/index.ts:107]: No, I don't intend to allow duplicates. Didn't read the doc, I thought it would lead to a better defined behavior.
